app/src/main/java/com/lagradost/cloudstream3/extractors/PlayLtXyz.kt

```diff
@@ -0,0 +1,84 @@
+package com.lagradost.cloudstream3.extractors
+
+import android.util.Log
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.lagradost.cloudstream3.app
+import com.lagradost.cloudstream3.utils.*
+import com.lagradost.cloudstream3.utils.AppUtils.tryParseJson
+
+class PlayLtXyz: ExtractorApi() {
+    override val name: String = "PlayLt"
+    override val mainUrl: String = "https://play.playlt.xyz"
+    override val requiresReferer = true
+
+    private data class ResponseData(
+        @JsonProperty("data") val data: String? = null
+    )
+
+    override suspend fun getUrl(url: String, referer: String?): List<ExtractorLink> {
+        val extractedLinksList = mutableListOf<ExtractorLink>()
+        //Log.i(this.name, "Result => (url) $url")
+        var idUser = ""
+        var idFile = ""
+        var bodyText = ""
+        val doc = app.get(url, referer = referer).document
+        //Log.i(this.name, "Result => (url, script) $url / ${doc.select("script")}")
+        doc.select("script").forEach {
```

Comment:
how about using something like 
`val bodyText =  doc.select("script").firstOrNull { ... } ?: ""`

Reply:
The first ``script`` does not always contain the required info so iterating over them is kinda needed. Though for optimization, I could skip the other elements once ``bodyText`` is not empty. Or, I could ``.filter{ }`` the ``doc.select``

Reply:
firstOrNull iterates over the list and stops the iteration once a result is found, i think this would be the equivalent code.

```kotlin
val bodyText = doc.select("script").firstOrNull {
    it?.toString()?.contains("var idUser") == true
}?.toString()
```

Reply:
Ah yeah that could work. Will change it